setup.py

```diff
@@ -45,7 +45,7 @@
 
 logger = logging.getLogger(__name__)
 
-version = '2.3.4.post5'
+version = '2.3.4.post6'
```

Comment:
Have you released this version of Airflow, or do you still need to do that?

Reply:
was holding off on that until there was merged